core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -20,6 +20,7 @@
 #include "refcount.h"
 #include "rights.h"
 #include "str.h"
+#include "ntdll.h" 
```

Comment:
ntdll.h header is not available on non-windows platforms, so I think it should be included conditionally